fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/DefaultPropertiesLoader.java

```diff
@@ -54,7 +54,8 @@
         OBJECT_STORE("fcrepo.object.directory"),
         BINARY_STORE("fcrepo.binary.directory"),
         MODE_INDEX("fcrepo.modeshape.index.directory"),
-        ACTIVE_MQ("fcrepo.activemq.directory");
+        ACTIVE_MQ("fcrepo.activemq.directory"),
+        ALLOWED_EXTERNAL_CONTENT("fcrepo.external.content.allowed");
```

Comment:
When this property is specified with JAVA_OPTS to point to a local file, the Tomcat start-up print statement is correct. However, when no system property is specified, the property name is printed (`fcrepo.external.content.allowed`) instead of the expected default location.

It appears that this class (`DefaultPropertiesLoader.java`) would need to be refactored to accommodate the new scenario presented by "external content allowed". If you would prefer the refactoring take place within its own ticket/PR, I am ok moving this through as-is.
As a note, such a separate ticket would likely be similar to a suggestion @whikloj recently made regarding consolidation of system properties.

Reply:
Hm yeah that is pretty weird looking and misleading given that there actually is not a default value. This seems like a potentially simple refactor, just add a "setDefaultValue" property for each enum value and have it false for this one case. I could do that refactor in this PR if that's all it is. But I don't recall @whikloj 's suggested changes for consolidating system properties, so that may complicate things.

Reply:
@awoods I have updated the behavior of `DefaultPropertiesLoader` to not set a default value for `fcrepo.external.content.allowed` and to not log the property when it is not set. Let me know if you want to keep this change or not.